src/KubeOps/Operator/Serialization/EntitySerializer.cs

```diff
@@ -9,10 +9,10 @@ internal class EntitySerializer
         private readonly ISerializer _yaml;
         private readonly JsonSerializerSettings _jsonSettings;
 
-        public EntitySerializer(ISerializer yaml, JsonSerializerSettings jsonSettings)
+        public EntitySerializer(ISerializer yaml, OperatorSettings operatorSettings)
```

Comment:
here we could inject the settings and use them accordingly
```suggestion
        public EntitySerializer(ISerializer yaml, JsonSerializerSettings serializerSettings)
```